openfisca_us/parameters/irs/credits/child_and_dep_care/phaseout/max.yaml

```diff
@@ -0,0 +1,7 @@
+description: Child & dependent care credit phaseout percentage rate ceiling
+metadata:
+  unit: /1
+values:
+  2013-01-01: 35.0
```

Comment:
```suggestion
  2013-01-01: 0.35
```
and below

Reply:
Thanks, fixed here and in the minimum parameter